.github/workflows/spec_test_on_nuttx.yml

```diff
@@ -75,20 +81,25 @@ jobs:
             mode: "-t aot",
             option: "CONFIG_INTERPRETERS_WAMR_AOT=y\\n"
           },
-          {
-            mode: "-t aot -X",
-            option: "CONFIG_INTERPRETERS_WAMR_AOT=y\\n"
-          },
-          {
-            mode: "-t classic-interp",
-            option: "CONFIG_INTERPRETERS_WAMR_CLASSIC=y\\n"
-          },
-          {
-            mode: "-t fast-interp",
-            option: "CONFIG_INTERPRETERS_WAMR_FAST=y\\n"
-          },
+          # {
+          #   mode: "-t aot -X",
+          #   option: "CONFIG_INTERPRETERS_WAMR_AOT=y\\n"
+          # },
+          # {
+          #   mode: "-t classic-interp",
+          #   option: "CONFIG_INTERPRETERS_WAMR_CLASSIC=y\\n"
+          # },
+          # {
+          #   mode: "-t fast-interp",
+          #   option: "CONFIG_INTERPRETERS_WAMR_FAST=y\\n"
+          # },
```

Comment:
@no1wudi 
why did you comment these out?

Reply:
There may have some potential bug, spectest for interpreter is extremly slow, about 4H.

Reply:
do you mean GC-related bugs?

Reply:
No, I think this bug is from test script, No-gc case also very slow, with GC enabled the test maybe over 6H that exceed the limit of GA runner